edgexfoundry/app-functions-sdk-go
# refactor!: Rename MakeItRun and MakeItStop APIs to Run and Stop

BREAKING CHANGE: MakeItRun renamed to Run and MakeItStop renamed to Stop







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

https://github.com/edgexfoundry/edgex-docs/pull/1060

## Testing Instructions
Run non-secure EdgeX Stack
Build and run app-template from this PR branch (`./app-new-service -cp -d`) with DEBUG logging enabled
Verify service bootstraps and starts processing events.

## New Dependency Instructions (If applicable)
